deploy/helm/listener-operator/templates/roles.yaml

```diff
@@ -76,9 +76,14 @@ rules:
       - ""
     resources:
       - nodes
+      - persistentvolumes
```

Comment:
For me the permissions seem a bit excessive. As you have access to OS, I trust you to make sure we need all of them :)